src/integration-test/groovy/au/org/ala/fieldcapture/OrganisationReportingSpec.groovy

```diff
@@ -0,0 +1,253 @@
+package au.org.ala.fieldcapture
+
+import com.icegreen.greenmail.util.GreenMail
+import com.icegreen.greenmail.util.GreenMailUtil
+import pages.Organisation
+import pages.ReportPage
+import pages.ViewReportPage
+import spock.lang.Shared
+
+import javax.mail.internet.MimeMessage
+
+class OrganisationReportingSpec extends StubbedCasSpec {
+
+    @Shared
+    GreenMail greenMail = new GreenMail()
+
+    def orgId = "test_organisation"
+
+    def setupSpec(){
+        useDataSet("dataset_mu")
+        greenMail.start()
+    }
+
+    def cleanup() {
+        waitFor {
+            logout(browser)
+        }
+        greenMail.stop()
+
+    }
+
+    def "Organisation reports are displaying in the reporting tab"() {
+
+        setup:
+        loginAsMeritAdmin(browser)
```

Comment:
You should not be using MERIT admin for tasks normally done by an organisation admin, otherwise these tests will miss an issue where we accidentally remove access to an org admin user.